README.md

```diff
@@ -1,24 +1,158 @@
-# README
+<a name="readme-top"></a>
 
```

Comment:
- [ ] I could not fetch the greeting to the frontend due to CORS policy blocking the origin of the request. 

  * In your readme file you need to uncomment line 37
    ```
    gem "rack-cors"
    ```
  * Then go to the `config\initializers\cors.rb` file and uncomment lines 8 to 16.
  * Change `origins "example.com"` to this `origins "*"`
  * Run `bundle install` and restart your server

  ![image](https://github.com/ghreza-crypto/hello-rails-back-end/assets/81468675/1d8e6985-cb32-4fa2-9d04-de2998e7eabd)